test/requests/is_api_request_test.rb

```diff
@@ -31,7 +33,9 @@ def app
   test "test upload via api" do
     filename = GemFactory.gem_file(:example)
     header "Accept", "text/plain"
-    post '/upload', { file: Rack::Test::UploadedFile.new(filename, 'application/octet-stream', true) }
+    silence_stream($stdout) do
+      post '/upload', { file: Rack::Test::UploadedFile.new(filename, 'application/octet-stream', true) }
+    end
```

Comment:
I could not see advantages by suppressing the output here, too.

Reply:
Same comment as above.